config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  namespace :admin do
+    resources :categories
+  end
+  resources :products
   scope "(:locale)", locale: /en|vi/ do
     get "sessions/new"
     get "sessions/create"
```

Comment:
Chỗ này đưa vào i18n lun he huynh
```suggestion
  scope "(:locale)", locale: /en|vi/ do
  namespace :admin do
    resources :categories
  end
  resources :products
    get "sessions/new"
    get "sessions/create"
```